lib/announce/adapters/shoryuken_adapter.rb

```diff
@@ -43,7 +44,8 @@ def register_class(worker_class)
         def active_job?
           defined?(::ActiveJob) &&
           defined?(ActiveJob::QueueAdapters::ShoryukenAdapter) &&
-          ActiveJob::Base.queue_adapter == ActiveJob::QueueAdapters::ShoryukenAdapter
+          (ActiveJob::Base.queue_adapter == ActiveJob::QueueAdapters::ShoryukenAdapter ||
+          ActiveJob::Base.queue_adapter.instance_of?(ActiveJob::QueueAdapters::ShoryukenAdapter))
```

Comment:
@agustin what was this change for? Are you setting the adapter to an instance for some reason, or did Rails 5 produce an error?